### PR TITLE
Update to PowerShell Sample Code after Running in the Dev Container

### PR DESCRIPTION
**Pull request by DilmurodMak**
Following Update Fixes following issues after testing it in the Dev Container: 

1. Added required installation for Dockerfile "Install-Module Az", It is required to run PowerShell CLI commands
2. Document file updated - It shows additional steps on how to authenticate to Azure from Docker Container. Since it sees it as another device
3. Typos in the code - File path for helper files had a typo. and bicep.psm1 had missing closing bracket

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -54,10 +54,10 @@
 
 	// Use 'postCreateCommand' to run commands after the container is created.
 	// Installs:
-	//   1. Mega Linter
+	//   1. Mega Linter`
 	//   2. Pester
 	//   3. Configures benchpress Python module
-	"postCreateCommand": "npm install -g mega-linter-runner && pwsh -command Install-Module -Name Pester -Force -SkipPublisherCheck && pip install --editable ./framework/python/",
+	"postCreateCommand": "npm install -g mega-linter-runner && pwsh -command Install-Module -Name Pester -Force -SkipPublisherCheck && pip install --editable ./framework/python/ && Install-Module Az",
```
Comment (omusavi) [inline .devcontainer/devcontainer.json:60]: The [docs](https://code.visualstudio.com/docs/devcontainers/create-dev-container#_rebuild) state that its more efficient for us to be adding these installes into the Dockerfile and not in the `postCreateCommand`, would you be able to make that change here?

**Review by omusavi — APPROVED** [review]

File: config/megalinter/.cspell.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "MSRC",
     "msrc",
     "Benchpress",
+    "BenchPress",
```
Comment (Rhernandez513) [inline config/megalinter/.cspell.json:32]: duplicate?

**Review by Rhernandez513 — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (Rhernandez513) [reply .devcontainer/devcontainer.json:60]: one or two post create command(s) is cool, 4 is too much

Comment (DilmurodMak) [reply .devcontainer/devcontainer.json:60]: I will touch base with Uffaz and do necessary changes

Comment (DilmurodMak) [reply .devcontainer/devcontainer.json]: After removing PowerShell commands into Dockerfile, docker image builds successfully but run commands in the docker file does not get executed after the image is built. Currently investigating it

Comment (omusavi) [reply .devcontainer/devcontainer.json]: Timebox investigation here to a couple hours, feel free to file an issue to fix this later, not required for merging this PR

File: config/megalinter/.cspell.json (hunk 2 above)
Comment (omusavi) [reply config/megalinter/.cspell.json:32]: I think cspell is case sensitive

**Pull request merged**